DataFrame.pc.in

```diff
@@ -0,0 +1,9 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/include
+
+Name: DataFrame
+Description: C++ DataFrame -- R's and Pandas DataFrame in modern C++ using native types, continuous memory storage, and no virtual functions.
+Version: @PROJECT_NAME_VERSION@
+Cflags: -I${includedir}
+Libs: -L${libdir} -lDataFrame -lrt
```

Comment:
Will this break/interfere with Windows build? Windows and MacOS don't have `rt` library.
In the CMake file I made sure `rt` is only for Linux and not for Windows and MacOS.

Reply:
I'v removed the `-lrt`. 
I have only Linux develop environment. And I test it on Linux only.